docs/asset-store/docs/06-01-asset.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Asset
+type: Custom Resource
+---
+
+The `assets.assetstore.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to define an asset to store in a cloud storage bucket. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```
+kubectl get crd assets.assetstore.kyma-project.io -o yaml
+```
+
+## Sample custom resource
+
+This is a sample resource (CR) that provides details of the bucket for storing assets.
+
+```
+apiVersion: assetstore.kyma-project.io/v1alpha1
+kind: Asset
+metadata:
+  name: my-package-assets
+  namespace: default
+spec:
+  source:
+    mode: single
+    url: https://some.domain.com/main.js
+  bucketRef:
+    name: my-bucket
+
+```
+
+### Validation and mutation webhook services
+
+You can also define validation and mutation services:
+- **Validation webhook** performs the validation of fetched assets before they are uploaded to the bucket. It can be a list of several different validation webhooks and all of them should be processed even if one fails. It can refer either to the validation of a specific file against a specification or to the security validation. The validation webhook returns the validation status.
+- **Mutation webhook** acts similarly to the validation service. The difference is that it mutates the asset instead of just validating it. For example, this can be an asset rewriting through the `regex` operation or `keyvalue`, or the modification in the JSON specification. The mutation webhook returns modified files instead of information on the status.
```

Comment:
```suggestion
- **Mutation webhook** acts similarly to the validation service. The difference is that it mutates the asset instead of just validating it. For example, this can mean asset rewriting through the `regex` operation or `keyvalue`, or the modification in the JSON specification. The mutation webhook returns modified files instead of information on the status.
```